frontend/src/employee-frontend/components/child-information/BackupPickup.tsx

```diff
@@ -84,11 +84,14 @@ function BackupPickup({ id }: BackupPickupProps) {
         clearUiMode()
       }
     }
+
     return (
       <FormModal
         title={i18n.childInformation.backupPickups.add}
-        reject={{ action: () => clearUiMode(), label: i18n.common.cancel }}
-        resolve={{ action: () => saveBackupPickup(), label: i18n.common.save }}
+        resolveAction={() => saveBackupPickup()}
+        resolveLabel={i18n.common.save}
+        rejectAction={() => clearUiMode()}
+        rejectLabel={i18n.common.cancel}
```

Comment:
You  could just pass `saveBackupPickup` and `clearUiMode` here.

Reply:
Good catch! I just used some macro trickery to convert them all without looking too much 😄